docs/security.md

```diff
@@ -258,3 +258,51 @@ If you want to use the command `curl`, you can follow the commands:
 openssl x509 -inform der -in $JAVA_HOME/localhost.crt -out certificate.pem
 curl -v -X GET --cacert ./certificate.pem -H "Accept: application/vnd.gravitino.v1+json" -H "Content-Type: application/json" https://localhost:8433/api/version
 ```
+
+## Custom filters
```

Comment:
I prefer to move doc to `GravitinoServer Configuration` because Custom filters is general and not limited to safety

Reply:
I refer to Spark. https://spark.apache.org/docs/latest/security.html

Reply:
I would suggest to move this part to server configurations, not here. Since user can configure other filters like redirect, it is not purely a security related feature.

Reply:
OK.